php/laravel/artisan

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env php
```

Comment:
do we really need this file in production ? (except for warming caches)

Reply:
The file isn't needed to run, but is required for caching. Not sure how to include them otherwise?

Reply:
I think we can remove it. Cache will be create on warm-up